### DISCUSSION TITLE
@prisma/client not working

### DISCUSSION THREAD

**Discussion by satvik-1203** [prisma/prisma#10330] [Q&A] [answered]
Version:  3.5.0
Error: `Invalid 'prisma.comment.findMany()' invocation:
Authentication failed against database server at 'db.__.supabase.co', the provided database credentials for 'postgres' are not valid.`

I'm able to connect to the server when using **Prisma studio** but it breaks when I try to connect it via code (`@prisma/client`)

**Comment by satvik-1203** [comment, marked as answer]
Alright, I fixed the error. IDK if it's Prisma or Postgres SQL problem but my Prisma client was working when I started using MySQL database. The reason I say it can be Prisma's bug is that, while using Prisma studio, It was working, able to add tables and stuff. but Prisma client wouldn't work, I also saw the same error with other people using Postgres SQL

**Reply by satvik-1203** [reply]
Nevermind, it was just my password being outside the range of UTF-8 and the URI couldn't encode it properly

**Reply by Zanadh** [reply]
> outside the range of UTF-8

@satvik-1203  how did you find out that was the problem? i think im having the same issue but i dont know how to verify it
is there any solution instead of changing the password?

**Reply by Zanadh** [reply]
> > outside the range of UTF-8
> 
> @satvik-1203 how did you find out that was the problem? i think im having the same issue but i dont know how to verify it is there any solution instead of changing the password?

Nevermind,  i just need to percent encode my password

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
